dev/devicelab/test/tasks/build_test_task_test.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_devicelab/framework/runner.dart';
+import 'package:flutter_devicelab/framework/task_result.dart';
+
+import '../common.dart';
+
+void main() {
+  final Map<String, String> isolateParams = <String, String>{
+    'runProcessCleanup': 'false',
+    'timeoutInMinutes': '1',
+  };
+
+  test('runs build and test when no args are passed', () async {
+    final TaskResult result = await runTask(
```

Comment:
@jmagman @jonahwilliams do you know if the isolates are expected to fail on windows? The tests are timing out on Windows from what looks like failing to start the isolate.

Reply:
Which failures do you mean? (generally Dart code works fine on all platforms)

Reply:
Sweet. Then I think the issue is this is the first time this code is being run in unit tests. It looks like flutter config was taking longer on windows than mac and linux, so I added an option to skip flutter config in the tests.

Reply:
`flutter config` should return really fast:
```
"flutter config" took 54ms.
```
If there's a legit reason for these to take longer than a minute, then you can increase the individual test timeout.  Though I'm suspicious.

In any case, `flutter config` only needs to be run once before all the tests are run, not once per test.  Might be better to move it into the test runner.
https://github.com/flutter/flutter/blob/90c66f5db3aa89fe88a7b77d8b7ee914ed5ed808/dev/bots/test.dart#L94-L103

Reply:
Should we be running flutter commands in the devicelab unit tests? I was running the devicelab/tests using `dart test`, and adding this to bots/test.dart would require using `dart bots/test.dart`

Reply:
> I was running the devicelab/tests using dart test

The devicelab tests are running `bots/test.dart` with `SHARD=framework_tests SUBSHARD=misc`, not the individual `dart test`s.  I don't see any `flutter config` taking more than 60 ms in those logs.